src/out_hoi4/characters/out_character.cpp

```diff
@@ -0,0 +1,163 @@
+﻿#include "src/out_hoi4/characters/out_character.h"
+
+#include <fstream>
+#include <ranges>
+
+#include "external/fmt/include/fmt/format.h"
+
+namespace
+{
+std::string OutputTraits(const std::set<std::string>& traits)
+{
+   std::string traits_string;
+   for (const auto& trait: traits)
+      traits_string += trait + " ";
```

Comment:
I bet it's possible to do this with fmt::format. The Range Format Specifications section [here](https://fmt.dev/latest/syntax.html) makes it look like it should be simple, but I haven't tried it.